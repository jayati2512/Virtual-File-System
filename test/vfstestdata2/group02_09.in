createvfs colors 120
mountvfs colors
makedir /colors primary
makedir /colors secondary
addfile /colors/primary red.txt /usr/include/poll.h
addfile /colors/primary green.txt /usr/include/syslog.h
addfile /colors/primary yellow.txt /usr/include/syscall.h
addfile /colors/secondary pink.txt /usr/include/termio.h
addfile /colors/secondary grey.txt /usr/include/ustat.h
addfile /colors/secondary orange.txt /usr/include/stab.h
deletedir 
deletedir /colors
deletedir /colors/primary
removefile /colors/primary/red.txt
removefile /colors/primary/green.txt
removefile /colors/primary/yellow.txt
deletedir /colors/primary
unmountvfs colors
