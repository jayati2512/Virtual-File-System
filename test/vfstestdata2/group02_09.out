createvfs_SUCCESS
mountvfs_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
addfile_SUCCESS
addfile_SUCCESS
addfile_SUCCESS
addfile_SUCCESS
addfile_SUCCESS
addfile_SUCCESS
deletedir_FAILURE VFS_INSUFFICIENT_ARGUMENTS
deletedir_FAILURE DIRECTORY_IS_NOT_EMPTY
deletedir_FAILURE DIRECTORY_IS_NOT_EMPTY
removefile_SUCCESS
removefile_SUCCESS
removefile_SUCCESS
deletedir_SUCCESS
unmountvfs_SUCCESS
